nautobot/extras/api/views.py

```diff
@@ -905,6 +905,16 @@ class TagViewSet(CustomFieldModelViewSet):
     serializer_class = serializers.TagSerializer
     filterset_class = filters.TagFilterSet
 
+    def get_serializer_class(self):
```

Comment:
Probably need to decorate the viewset with some `extend_schema_view` code for correct API documentation, similar to what was done in the nat_outside PR.

Reply:
See: https://github.com/nautobot/nautobot/pull/1420/files#diff-449b75151a29c86fd2fcd1ba1ae9a79e8f2d2e11aeb1affc24bc55bb16a6b22aR295-R304